integr8ly/integreatly-operator
# MGDAPI-4711 bump observability-operator 3.0.13 to 3.0.16

# Issue link


# What
This PR bumps the version of the observability-operator from v3.0.13 to v3.0.16.

# Verification steps

- Provision an OSD cluster
- Prepare the cluster and install RHOAM from the latest stable version `v1.27.0`:
```
git checkout tags/rhoam-v1.27.0 -b rhoam-v1.27.0

make cluster/prepare/local LOCAL=false

cat << EOF | oc create -f -
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: rhoam-operators
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: quay.io/tdimov0/managed-api-service-index:1.27.0
EOF

make deploy/integreatly-rhmi-cr.yml LOCAL=false
```
- Install the RHOAM operator from Operator Hub
- Installation should complete successfully. Confirm that the installed version of the observability-operator is `v3.0.13`:
```
oc get ClusterServiceVersion -n redhat-rhoam-observability-operator observability-operator.v3.0.13
```
- Patch the catalog source to point at `1.27.1` and upgrade RHOAM:
```
oc patch catalogsource rhoam-operators -n openshift-marketplace --type=json -p='[{"op": "replace", "path": /spec/image, "value": "quay.io/tdimov0/managed-api-service-index:1.27.1"}]'
```
- After a few minutes, verify that the installed version of the observability-operator is now `v3.0.16`:
```
oc get ClusterServiceVersion -n redhat-rhoam-observability-operator observability-operator.v3.0.16
```
